Pod/Classes/WPMediaCollectionViewCell.m

```diff
@@ -88,16 +88,15 @@ - (void)setAsset:(id<WPMediaAsset>)asset {
     CGSize requestSize = CGSizeApplyAffineTransform(self.frame.size, CGAffineTransformMakeScale(scale, scale));
 
     requestKey = [_asset imageWithSize:requestSize completionHandler:^(UIImage *result, NSError *error) {
-        BOOL animated = ([NSDate timeIntervalSinceReferenceDate] - timestamp) > ThredsholdForAnimation;
         if (error) {
             self.image = nil;
-            NSLog(@"%@", [error localizedDescription]);
             return;
         }
         // Did this request changed meanwhile
         if (requestKey != self.tag) {
             return;
         }
+        BOOL animated = ([NSDate timeIntervalSinceReferenceDate] - timestamp) > ThredsholdForAnimation;
```

Comment:
Small typo here (not introduced in this PR, but nice to fix!): `ThredsholdForAnimation`